onnxruntime/core/providers/cuda/shared_inc/fpgeneric.h

```diff
@@ -25,12 +25,14 @@ inline cublasStatus_t cublasGemmHelper(cublasHandle_t handle, cublasOperation_t
   return cublasDgemm(handle, transa, transb, m, n, k, alpha, A, lda, B, ldb, beta, C, ldc);
 }
 inline cublasStatus_t cublasGemmHelper(cublasHandle_t handle, cublasOperation_t transa, cublasOperation_t transb, int m, int n, int k, const half* alpha, const half* A, int lda, const half* B, int ldb, const half* beta, half* C, int ldc) {
+  cublasSetMathMode(handle, CUBLAS_TENSOR_OP_MATH);
```

Comment:
>CUBLAS_TENSOR_OP_MATH [](start = 28, length = 21)

TensorCore is for Volta only, so move it inside if?

Reply:
Set the flag CUBLAS_TENSOR_OP_MATH will impact other cuda ops. I suggest to use similar way as CublasConfigHelper in Attention that reset it to default once GEMM is done.